src/main/java/interfaces/IFileIO.java

```diff
@@ -5,6 +5,8 @@
   public String workingDirectory();
   public byte[] getContent(String fileName) throws IOException;
   public String[] getFiles(String directory) throws IOException;
+  public void writeContent(String fileName, byte[] body) throws IOException;
```

Comment:
Ever heard of the I in SOLID?

This interface reminds of that :) Although these are exclusively file/directory operators.

Can you answer the question "who owns this interface"?